test.sh

```diff
@@ -20,3 +20,20 @@ mkdir ref && mv ref*.fa.gz ref/
 ( cd ref && bowtie2-build -q ref1.fa.gz ref1 && bowtie2-build -q ref2.fa.gz ref2 )
 
 minute run "$@"
+
+# Run a standard test
+cd ..
+rm -rf data_std
+rm -rf testrun_std
+
+mkdir data_std && cd data_std
+tar --strip-components=1 -xf ../${testdata_file}
+mkdir fastq && mv -- testdata-*.fastq.gz fastq/
+
+cd ..
+minute init testrun_std --reads ./data_std/fastq --barcodes ./testdata/barcodes.tsv --input testdata-Inp
```

Comment:
Relative paths don’t need a `./` in the beginning.
```suggestion
minute init testrun_std --reads data_std/fastq --barcodes testdata/barcodes.tsv --input testdata-Inp
```